lib/lita/handlers/puppet.rb

```diff
@@ -136,6 +145,31 @@ def node_profiles(response)
         end
       end
 
+      def nodes_with_class(response)
+        puppet_class = response.matches[3]
```

Comment:
This won't work. `response.matches` returns an Array of an Array (no idea why, but it comes from [`String#scan()`](https://ruby-doc.org/core-2.2.0/String.html#method-i-scan). Try testing your regex from the route definition above against a mock IM and you'll see what I mean. This should probably be `response.matches[0][3]`, since we want the item at index 3 from the Array returned by `response.matches[0]`.